### ISSUE TITLE
import_map.json and lint don't work well together

### ISSUE THREAD

**Issue by kecrily** [denoland/deno_lint#784]
### Lint Name

import-prefix-missing

### Code Snippet

```ts
import { Command } from "cliffy";
```

import_map

```json
{
  "imports": {
    "cliffy": "https://deno.land/x/cliffy@v0.19.4/mod.ts"
  }
}
```

### Expected Result

Not subject to import-prefix-missing

### Actual Result

Relative import path "clippy" not prefixed with / or ./ or ../ from “........”

### Additional Info

This happens when only a typescript file is imported, and prompt for error in VSCode only.

### Version

```
$ deno --version
deno 1.12.2 (release, x86_64-pc-windows-msvc)
v8 9.2.230.14
typescript 4.3.5
```

**Comment by kecrily** [comment]
Maybe I should go to `vscode_deno` and submit an issue.
